types/ityped/index.d.ts

```diff
@@ -1,15 +1,22 @@
-// Type definitions for ityped 0.0
+// Type definitions for ityped 1.0
 // Project: https://github.com/luisvinicius167/ityped
 // Definitions by: Daniel Rosenwasser <https://github.com/DanielRosenwasser>
```

Comment:
Any interest in adding yourself as a maintainer @MaySoMusician? I'm not as involved with using this library.

Reply:
I was wondering that line is automatically updated, or needs to be edited by hand. I'll add my name there, although this library has had no update for last 2 years